recipes/openai-whisper/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "openai-whisper" %}
+{% set version = "20240930" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/openai-whisper-{{ version }}.tar.gz
+  sha256: b7178e9c1615576807a300024f4daa6353f7e1a815dac5e38c33f1ef055dd2d2
+
+build:
+  entry_points:
+    - whisper=whisper.transcribe:cli
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - numba
+    - numpy
+    - pytorch  # [not linux64]
+    - pytorch =*=cuda*  # [linux64]
```

```suggestion
    - pytorch
```

I _think_ this can be done. This will then fail on Linux if Cuda is unavailable and succeed if Cuda is available. 

If I remove it, I'm pretty sure it will not run the Cuda tests in the CI, so I cannot verify this and need an expert's help.

---

> If I remove it, I'm pretty sure it will not run the Cuda tests in the CI, so I cannot verify this and need an expert's help.

There is no Azure CI runner with actual GPUs.